main.js

```diff
@@ -4,20 +4,20 @@ const active = require('./lib/active');
 const get = require('./lib/get');
 const dynamos = require('./lib/dynamos');
 const health = require('./lib/health');
-const cache = require('./lib/cache');
 
 let metrics;
-let useCache = false;
 let timeout;
 
 exports.health = health.check;
 
 exports.get = fromURL => {
-	if(useCache){
-		let cacheItem = cache.retrieve(fromURL);
-		if(cacheItem){
-			return Promise.resolve(cacheItem);
-		}
+	if (fromURL[fromURL.length - 1] === '/') {
```

Comment:
hehehehe, this also needed to check that the URL wasn't *just* `/` because otherwise put the front page into an infinite loop 😉